packages/react/src/combobox/ComboboxMulti.tsx

```diff
@@ -84,8 +85,40 @@ export function ComboboxMulti<Option extends DefaultComboboxOption>({
 		[options, inputValue, selectedItems]
 	);
 
+	const previousSelectedItemsRef = useRef<Option[]>([]);
+
 	const multiSelection = useMultipleSelection({
 		selectedItems,
+		getA11yStatusMessage: (state) => {
+			const { selectedItems } = state;
+
+			// No changes made or an item was added, nothing to announce
+			if (
+				selectedItems.length === previousSelectedItemsRef.current.length ||
+				selectedItems.length > previousSelectedItemsRef.current.length
+			) {
+				previousSelectedItemsRef.current = selectedItems;
+				return '';
```

Comment:
I would argue that having an item added is useful info, since the next announcement otherwise just the next item in the list being "selected"

Reply:
We can add that in if you want. I haven't been able to find out what JAWS etc announce or why this was the default for `downshift`. Shall we pop it in and run it through a11y testing?

Reply:
Yeah, sounds like a good idea, it's easy enough to remove if they have an issue with it in re-testing

Reply:
- [ ] Announce selection